modules/ingest-common/src/test/java/org/opensearch/ingest/common/IngestCommonModulePluginTests.java

```diff
@@ -36,6 +36,27 @@ private void runAllowlistTest(List<String> allowlist) throws IOException {
         }
     }
 
+    public void testInvalidAllowList() throws IOException {
```

Comment:
Is this test case different from `testAllowlistHasNonexistentProcessors`?

Reply:
Its not. I didnt realize there was one. 
I'll remove it!